angular-ngrx-scss/src/app/shared/components/filter-dropdown/filter-dropdown.component.scss

```
@@ -9,18 +9,23 @@
     display: inline-flex;
     align-items: center;
     border-radius: 0.375rem;
-    border: 1px solid variables.$gray300;
+    border: 1px solid variables.$gray100;
     background-color: variables.$gray100;
-    padding: 0.375rem 1rem;
+    padding: 0.5rem 1.25rem;
     line-height: 1.25rem;
-    color: variables.$gray900;
+    color: variables.$gray600;
+    font-size: 1rem;
+
+    &.isRepo {
```

```suggestion
    &--repo {
```
I see that in some places we use BEM and in other it seems we don't. It would be nice to unify that. But again, probably out of scope.

Anyway, since we use BEM conventions in this part of code, I think a BEM modifier would be appropriate here.

---

changed in both places.